src/EnergyPlus/HVACVariableRefrigerantFlow.hh

```diff
@@ -1049,7 +1088,200 @@ namespace HVACVariableRefrigerantFlow {
 		Real64 const Te, // Outdoor unit evaporating temperature
 		Array1< Real64 > const & Par // Array of parameters
 	);
-
+	
+	void
+	VRFOU_TeTc(
+		int const VRFCond, // Index to VRF outdoor unit
```

Comment:
And I just found them.   All of these functions.  Basically every function that has a first index of VRFCond.  These should be moved into member functions of the VRF class/struct.  So here's what I would love, and it's identical to what we were doing in the plant refactor:
 - Take VRFCond index out as an argument in both the hh and cc files
 - In the cc file, inside the function, each time you use `VRF( VRFCond ).` or whatever the variable name is, change it to `this->` (we can clean this up later, naysayers...)
 - Move this function definition inside the class/struct above here in the hh file
 - In the cc file, change the definition so it is: `VRFCondenserEquipment::VRFOU_TeTc`

If you did that for all of these functions, it would be a simple, but big step forward in a more structured VRF code base.  After that is complete, we can take a higher pass and decide if we want to change some ownership/structure to a more OO basis or not.